apps/evm/src/containers/Layout/useGetMenuItems.tsx

```diff
@@ -12,7 +12,6 @@ const useGetMenuItems = () => {
   const { stakedEthPoolComptrollerContractAddress } = useGetChainMetadata();
   const swapRouteEnabled = useIsFeatureEnabled({ name: 'swapRoute' });
   const historyRouteEnabled = useIsFeatureEnabled({ name: 'historyRoute' });
-  const convertVrtRouteEnabled = useIsFeatureEnabled({ name: 'convertVrtRoute' });
```

Comment:
We can remove the "convertVrtRoute" feature flag too then

Reply:
We actually can't, because it is still used for the route itself, which is only valid in BSC mainnet/testnet

Reply:
Ahh right, makes sense